lints/lint_rsa_public_exponent_not_in_range.go

```diff
@@ -27,12 +27,16 @@ func (l *rsaParsedTestsExpInRange) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rsaParsedTestsExpInRange) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	pubKey := c.PublicKey.(*rsa.PublicKey).E
+	key, ok := c.PublicKey.(*rsa.PublicKey)
+	if !ok {
+		return ResultStruct{Result: Error}, nil
+	}
+	exponent := key.E	
 	lowerBound := 65536 // 2^16 + 1
 	var upperBound big.Int
 	upperBound.Exp(big.NewInt(2), big.NewInt(256), nil)
 
-	if pubKey > lowerBound && upperBound.Cmp(big.NewInt(int64(pubKey))) == 1 {
+	if exponent > lowerBound && upperBound.Cmp(big.NewInt(int64(exponent))) == 1 {
```

Comment:
This is stated in CAB BR 6.1.6
RSA: The CA SHALL confirm that the value of the public exponent is an odd number equal to 3 or more.
Additionally, the public exponent SHOULD be in the range between 2^16+1 and 2^256‐1